### ISSUE TITLE
在做三分类问题的时候遇到了expected dense_1 to have 2 dimensions, but got array with shape (32, 1, 3)

### ISSUE THREAD

**Issue by xxxxxxxxy** [CyberZHG/keras-bert#87]
将y值变成了二值类别矩阵
`np_utils.to_categorical(y, NB_CLASSES)`
添加了Dense层
`p = Dense(NB_CLASSES, activation='softmax', input_shape=(3,))(x)`
模型编译
```
model.compile(
    loss='categorical_crossentropy',
    optimizer=Adam(1e-5), 
    metrics=['accuracy']
)
```
请问三分类问题后面对于这种二值类别矩阵应该如何处理呢？

**Comment by CyberZHG** [comment]
```python
np_utils.to_categorical(np.squeeze(y, axis=-1), NB_CLASSES)
```

**Comment by xxxxxxxxy** [comment]
问题解决了，谢谢你
